fa25-team-b/README.md

```
@@ -53,71 +56,275 @@ Continuing the work of students in the **Summer of 2025**, we are focusing on id
+---
+
+## 🔐 4. Google OAuth Setup
+
+Our data is located in a shared google folder, and is confidential. Since we cannot have our data shared publicly, we are using **Google OAuth** to access data from our drives locally. This works for anyone who has sharing access to the dataset folder, as long as you obtain the necessary JSON from a team member.
+
+### 4.1 Get JSON setup
+
+- Get the oauth_client.json file from a team member
+- Save it to:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)